### ISSUE TITLE
UnsupportedAudioFileException on exported jar

### ISSUE THREAD

**Issue by PixelDoted** [goxr3plus/java-stream-player#74]
java-stream-player version: 10.0.2

I can play MP3's in VSCode  
but not in an exported jar  

when I try to play an mp3 it outputs
```
Mar 31, 2022 11:32:36 AM com.goxr3plus.streamplayer.stream.StreamPlayer initAudioInputStream
INFO: File of unsupported format

javax.sound.sampled.UnsupportedAudioFileException: File of unsupported format
	at java.desktop/javax.sound.sampled.AudioSystem.getAudioInputStream(AudioSystem.java:1066)
	at com.goxr3plus.streamplayer.stream.FileDataSource.getAudioInputStream(FileDataSource.java:29)
	at com.goxr3plus.streamplayer.stream.StreamPlayer.initAudioInputStream(StreamPlayer.java:338)
	at com.goxr3plus.streamplayer.stream.StreamPlayer.open(StreamPlayer.java:291)
	at me.pixeldots.imaudio.Audio.AudioPlayer.playAudio(AudioPlayer.java:49)
	at me.pixeldots.imaudio.Audio.APHelper.lambda$0(APHelper.java:17)
	at java.base/java.lang.Thread.run(Thread.java:833)
```

I found this issue (#73)  
saying they were missing dependencies, but didn't say what dependencies  

I also tried the demo code but it still didn't work when exported to a jar

**Comment by goxr3plus** [comment]
Let's see if @omegaui can help you with this . If not i will have a look . Just yesterday i updated [XR3Player](https://github.com/goxr3plus/XR3Player) to Java 18 which is using java-stream-player so i will test it too why that may happen . Probably the exported jar is missing something . You can have a look on XR3Player pom file to see how i create the final jar file.

**Comment by omegaui** [comment]
@goxr3plus 
@PixelDoted 

When I came across this issue, I took a look at META-INF's services files of each jar dependency of java-stream-player and found out that some files have the same name and in preparing the fat jar they were overwriting each other therefore causing the UnsupportedAudi...... Exception. 


Thus, running from the IDE was preventing the overwrite as all dependencies were loaded dynamically.

So for me, dynamically loading the dependencies fixed the issue.
